src/pages/uxp/reference-js/Global Members/HTML DOM/Text.md

```diff
@@ -1,25 +1,35 @@
----
-jsDoc: true
----
 
 <a name="text" id="text"></a>
 
 # window.Text
 **See**: https://developer.mozilla.org/en-US/docs/Web/API/Text  
 
 
+<a name="new-text-new" id="new-text-new"></a>
+
+## Text(document, textContent)
+Creates an instance of Text.
+
+
+| Param | Type |
+| --- | --- |
+| document | `*` | 
+| textContent | `*` | 
```

Comment:
Are we sure we don't have types for this? Maybe `document: Document` and `textContent: string`? 🤔

Reply:
🟡

Reply:
Tracked in #75